lib/Agrammon/TechnicalParser.pm6

```diff
@@ -31,7 +31,14 @@ sub parse-technical(Str $to-parse) is export {
 }
 
 sub load-technical(IO::Path $model-path, Str $technical-file) is export {
-    my $tech-input = $model-path.IO.&child-secure($technical-file // 'technical.cfg');
+    my $tech-input;
+    # handle absolut path
+    if $technical-file and $technical-file ~~ / ^ '/' / {
+        $tech-input = $technical-file;
+    }
```

Comment:
Hm, but this means somebody can submit an absolute path via the web API also and bypass the `child-secure` below?

Reply:
You're right. As I don't see an easy way to handle absolute paths, I just reverted this change, so that I can at least merge the remaining fix.